vllm/transformers_utils/tokenizer_group/ray_tokenizer_group.py

```diff
@@ -0,0 +1,168 @@
+import asyncio
+import os
+from typing import List, Optional
+
+from transformers import PreTrainedTokenizer
+
+from vllm.config import TokenizerPoolConfig
+from vllm.lora.request import LoRARequest
+from vllm.engine.ray_utils import ray
+from vllm.transformers_utils.tokenizer_group.base_tokenizer_group import (
+    BaseTokenizerGroup)
+from vllm.transformers_utils.tokenizer_group.tokenizer_group import (
+    TokenizerGroup)
+from ray.util.scheduling_strategies import NodeAffinitySchedulingStrategy
+
+
+def _carry_over_env_vars_to_runtime_env(runtime_env: dict) -> dict:
```

Comment:
nit: helper methods can be classmethods (scoped to class) or at bottom of file (not the first thing you see when you open a file)